koji_wrapper/wrapper.py

```diff
@@ -62,9 +69,12 @@ def srpm_url(self, nvr=None):
             # errors here.
             raise inst
 
-    def _build_srpm_url(rpm=None, build=None):
+    def _build_srpm_url(self, rpm=None, build=None):
```

Comment:
This is the key fix in here, the rest is adding unittests and sanity around this same function